pkg/chartverifier/checks/checks.go

```diff
@@ -274,6 +281,31 @@ func ImagesAreCertified(opts *CheckOptions) (Result, error) {
 	return r, nil
 }
 
+func NoMissingRequiredAnnotations(opts *CheckOptions) (Result, error) {
+	c, _, err := LoadChartFromURI(opts.URI)
+	if err != nil {
+		return NewResult(false, err.Error()), err
+	}
+
+	if c.Metadata == nil {
```

Comment:
For chart testing we allow a use to specify extra values using flags such as --chart-set, would like to discuss whether we should allow annotations to be set using the same mechanism.